README.md

```diff
@@ -6,7 +6,7 @@ A repository for team interaction, syncing, and handling meeting notes across th
 Weekly team meetings take place on the [Jupyter Zoom room](https://zoom.us/my/jovyan?pwd=c0JZTHlNdS9Sek9vdzR3aTJ4SzFTQT09) on Wednesdays at 9:00 AM Pacific Time ([your time](https://www.thetimezoneconverter.com/?t=9%3A00%20am&tz=San%20Francisco)). Minutes will be taken at [Hackmd.io](https://hackmd.io/Y7fBMQPSQ1C08SDGI-fwtg). Each meeting's minutes will be added as a comment to an issue in this project.
 
 ## Weekly Jupyter triage meeting
-Weekly triage meetings take place on the [Jupyter Zoom room](https://zoom.us/my/jovyan?pwd=c0JZTHlNdS9Sek9vdzR3aTJ4SzFTQT09) on Thursdays at 10:00 AM Pacific Time ([your time](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=San%20Francisco)). We look at [JupyterLab issues that need triage](https://github.com/jupyterlab/jupyterlab/labels/status%3ANeeds%20Triage) and [Jupyter Notebook issues](https://github.com/jupyter/notebook/issues/). Minutes will be taken at [Hackmd.io](https://hackmd.io/HaQ3S_nPSbqaRtk9rj59iA). Each meeting's minutes will be added as a comment to an issue in this project.
+Weekly triage meetings take place on the [Jupyter Zoom room](https://zoom.us/my/jovyan?pwd=c0JZTHlNdS9Sek9vdzR3aTJ4SzFTQT09) on Tuesdays at 10:00 AM Pacific Time ([your time](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=San%20Francisco)). We look at [JupyterLab issues that need triage](https://github.com/jupyterlab/jupyterlab/labels/status%3ANeeds%20Triage) and [Jupyter Notebook issues](https://github.com/jupyter/notebook/issues/). Minutes will be taken at [Hackmd.io](https://hackmd.io/HaQ3S_nPSbqaRtk9rj59iA). Each meeting's minutes will be added as a comment to an issue in this project.
```

Comment:
Thanks for opening this! The time has also changed.

```suggestion
Weekly triage meetings take place on the [Jupyter Zoom room](https://zoom.us/my/jovyan?pwd=c0JZTHlNdS9Sek9vdzR3aTJ4SzFTQT09) on Tuesdays at 9:00 AM Pacific Time ([your time](https://www.thetimezoneconverter.com/?t=9%3A00%20am&tz=San%20Francisco)). We look at [JupyterLab issues that need triage](https://github.com/jupyterlab/jupyterlab/labels/status%3ANeeds%20Triage) and [Jupyter Notebook issues](https://github.com/jupyter/notebook/issues/). Minutes will be taken at [Hackmd.io](https://hackmd.io/HaQ3S_nPSbqaRtk9rj59iA). Each meeting's minutes will be added as a comment to an issue in this project.
```

Reply:
I agree with your suggested change.  I'm new to Github and got confused about how to accept (commit?) your change along with mine.  In addition was unsure about "resolving" the conversation.  I think I've accepted your change and they are ready to be implemented.